src/averageAges.js

```diff
@@ -55,7 +71,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    // eslint-disable-next-line max-len
+    ? people.filter(person => person.sex === 'm' && people.some(mother => mother.name === person.mother))
```

Comment:
Remove the eslint-disable comment and break the line properly to avoid exceeding the max line length